planner/core/fragment.go

```diff
@@ -54,6 +56,14 @@ type Fragment struct {
 	singleton bool // indicates if this is a task running on a single node.
 }
 
+type CTEGroupInFragment struct {
+	CTEStorage *PhysicalCTEStorage
+	CTEReader  []*PhysicalCTE
+
+	StroageTasks     []*kv.MPPTask
```

Comment:
```suggestion
	StorageTasks     []*kv.MPPTask
```
ditto for the below one